packages/blade/src/components/Table/Table.native.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import { Text } from '~components/Typography';
+import type { SurfaceLevels } from '~tokens/theme/theme';
+
+export type TableNode<Item> = Item & {
+  id: string;
+};
+
+export type TableData<Item> = {
+  nodes: TableNode<Item>[];
+};
+
+export type TableProps<Item> = {
```

Comment:
Let's have types.ts file and put all the reusable types there? So you don't have to copy these types for RN file again?

Reply:
Done